apps/web/src/api/minutes/schema.ts

```diff
@@ -0,0 +1,16 @@
+import {z} from "zod";
+
+export const minuteSchema = z.object({
+  _id: z.string(),
+  date: z.string(),
+  allmote: z.boolean(),
+  title: z.string(),
+  document: z
+    .object({
+      asset: z.object({
+        url: z.string(),
+      }),
+    })
+    .transform((m) => m.asset.url),
```

Comment:
```suggestion
  document: z.string(),
```
Siden det ikke blir et objekt lengre hvis vi endrer queryen